src/agentlab/llm/llm_utils.py

```
@@ -166,13 +165,13 @@ def retry_raise(
         messages.append(answer)  # TODO: could we change this to not use inplace modifications ?
 
         try:
-            return parser(answer.content)
+            return parser(answer.get("content"))
```

perhaps better to use answer["content"] to avoid silent fail